### ISSUE TITLE
dask read_csv ignore broken files 

### ISSUE THREAD

**Issue by wybert** [dask/dask#9529]
I try to read the CSV files with,
```python
import dask.dataframe as dd
data = dd.read_csv("*.csv.gz",sep='\t',dtype='unicode', low_memory='true',compression='gzip',blocksize=None)
```
I have some CSV files which are broken files. I want to ignore that. It will be great if we have some function like below,

```python
import dask.dataframe as dd
data = dd.read_csv("*.csv.gz",sep='\t',dtype='unicode', low_memory='true',compression='gzip',blocksize=None,ignore_broken_files = True)
```
Which will skip the files that can't be read.

**Comment by GenevieveBuckley** [comment]
Can you provide more details about how your csv files are broken, and what kind of error message you are getting? A [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) could be helpful here.

**Comment by martindurant** [comment]
Labeling this as a feature request - it seems that the OP doesn't want to fix the broken files, but not process them at all. It would require some development to pass the argument to the loader and try/except all/specific errors.

**Comment by wybert** [comment]
> Can you provide more details about how your csv files are broken, and what kind of error message you are getting? A [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) could be helpful here.

Hi, it's some broken files we don't need it but these among many CSV files. We could just use try-except to ignore that but I am wondering if Dask can do it automatically.

**Comment by martindurant** [comment]
No, I don't believe that is automatically done by dask for you.
